searx/static/themes/simple/gruntfile.js

```diff
@@ -138,6 +138,12 @@ module.exports = function (grunt) {
             nonull: true,
             filter: file_exists,
           },
+          {
+            src: ['src/less/custom.less'],
+            dest: 'css/custom.css',
+            nonull: true,
+            filter: file_exists,
```

Comment:
I can't see the benefit of an additional CSS file: `custom.css` is built file like the other built files => `custom.less` won't be customize because it requires to build the themes, so `git pull` is not going to work.

So, despite what I've said before, and after a second thought, I suggest to make it simple:
* rename custom.less to embedded.less
* move the CSS rules about embedded media from style.less to embedded.less ( `.embedded-content iframe {`, `.audio-control audio {`, etc...)
* include embedded.less in the normal build process (if it is not already done).
* leave the customization for later.

So everything about embedded media is in one place.

Reply:
> include embedded.less in the normal build process (if it is not already done).

A ``@import "embedded.less"`` in `searx/static/themes/simple/src/less/style.less` is needed  .. see 8acf540f